### ISSUE TITLE
False "start time must precede end time" error message on OBS recorded MKV file

### ISSUE THREAD

**Issue by anjari** [mifi/lossless-cut#875]
**Describe the bug**
Clicking the "Set Cut Start" & "Set Cut End" buttons on MKV file could show error message pop-up "Start time must precede end time" despite the start & end time value are correct.

The MKV file was a recording video file result from OBS. The MKV file link is at [here](https://www.mediafire.com/file/h8i8f6mcfvj74km/test1.mkv/file
).

**To Reproduce**
1. Open the MKV file.
2. Click on "Set cut start to current position" button (the one with hand pointing to left icon). You don't need to change the time value.
3. Click on "Set cut end to current position" button (the one with hand pointing to right icon). You don't need to change the time value. But if you make a change on the cut start/end time, make sure the time values are correct, i.e. the end time should be bigger than the start time.
4. The error message pop-up "Start time must precede end time" will show up. If it doesn't show up, try to change the start/end time and retry the step 2 & 3.

**Expected behavior**
The  error message pop-up shouldn't show up.

**Desktop (please complete the following information):**
 - OS: Windows 7 64 bit.
 - LosslessCut version: 3.39.

**Comment by mifi** [comment]
Hi.
Im not sure if I understand correctly, but if you
1. open a file
2. Set cut start (time 00:00:00.000)
3. Set cut end (also time 00:00:00.000)

…then I believe it is correct that you get this error because the length of the resulting segment would be 0 which is not allowed.

or am I misunderstnnding?

**Comment by anjari** [comment]
> 3\. Set cut end (also time 00:00:00.000)

Nope, the default for the cut end time is set to the last time value if you just opened the file. Also, I had said "...despite the start & end time value are correct", which means the end time is larger than the start time.

**Comment by mifi** [comment]
Ok, no worries. thanks for clarifying what was confusing.
